gowin/arch.cc

```diff
@@ -1258,6 +1258,28 @@ static bool skip_aux_oser16(std::string device, int row, int col)
     return false;
 }
 
+WireId Arch::get_make_port_wire(const DatabasePOD *db, const BelsPOD *bel, int row, int col, IdString port)
+{
+    IdString wirename = IdString(pairLookup(bel->ports.get(), bel->num_ports, port.hash())->src_id);
+    IdString wire = idf("R%dC%d_%s", row + 1, col + 1, wirename.c_str(this));
+    if (wires.count(wire) == 0) {
```

Comment:
`if (!wires.count(wire))` is more idiomatic across the rest of the nextpnr codebase

Reply:
done